### ISSUE TITLE
Character counter appearance doesn't support minimal labels

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3233]
I don't think this should have a high priority, but we discovered this while testing 2017.1 (see also #3213).

`fr|character-counter` has the `lhha` mode so is in charge of handling labels. The nested field doesn't have a label, and instead we use `xxbl:label-for`.

Should XBL components have an option to say they support minimal labels? In which case the component would not output the label, and instead the component would have a chance to do what is needed.

Q: What would the component need to handle the minimal label from the inside?

**Comment by ebruchez** [comment]
Similar for `fr:number`, see #3277.
